README.md

```diff
@@ -11,7 +11,7 @@ The implementation includes an upgradable modular account with two plugins (`Sin
 
 ## Development
 
-Anyone is welcome to submit feedback and/or PRs to improve code or add Plugins.
+Anyone is welcome to submit feedback and/or PRs to improve code or add Modules.
```

Comment:
Minor note/todo: we should consider if having modules be on this repo is the way to go, or if we should split them out into another repo (long term)

Reply:
Yeah I think we've already started encouraging people to put these in their own repos or in https://github.com/erc6900/sample-plugins, to make the maintenance load easier. The README has some other out-of-date things, too.

Reply:
Yeah, gonna remove the add modules part.